TODO

```diff
@@ -1,10 +1,7 @@
-provider TODO -
-  - password should be a property and not a param
-     - what if we tried to log in as that user?
+provider TODO:
   - can permissions from list_user_permissions contain whitespace?
-  - what about defaultfor :true?
-  - prefetching for performance
+	- https://github.com/puppetlabs/puppetlabs-rabbitmq/blob/master/lib/puppet/type/rabbitmq_user_permissions.rb#L54 Should I go ahead and disallow whitespace?
+  - what about default for :true?
+	- What does this mean?
   - rabbit plugin should require rabbitmq class
-  - rabbitmq class should be renamed server??
-  - service name should default to -server
-  - cannot find stomp package
+	- Is this still the case?
```

Comment:
I think this takes care of that https://github.com/puppetlabs/puppetlabs-rabbitmq/blob/master/manifests/init.pp#L241